l10n_ch_payment_slip/models/invoice.py

```diff
@@ -158,7 +158,7 @@ def _check_bvr_generatable(self):
                     msg.append(_('The bank account {} used in invoice has no '
                                  'BVR/ESR adherent number.'
                                  ).format(bank_acc.acc_number))
-                if not bank_acc.acc_type != 'postal' or not bank_acc.ccp:
+                if not bank_acc.acc_type == 'postal' or not bank_acc.ccp:
```

Comment:
Okay this is not exactly what we want.

Before the fix: only bank account with ccp works.
After the current fix only account of type postal will work.

Here are the 3 use cases:

* A: bank account without ccp -> raise an error
* B: bank account with ccp -> OK
* C: account type postal (thus always with ccp) -> OK

Only A must fail.

Thus the condition must be 

`bank_acc != 'postal and not bank_acc.ccp'`

unless you prefer:

`not (bank_acc.acc_type == 'postal' or bank_acc.ccp)`

Or even we could simplify it to `not bank_acc.ccp` as ccp should be computed based on account_number if the type is postal anyway.
https://github.com/OCA/l10n-switzerland/blob/9.0/l10n_ch_base_bank/models/bank.py#L187